src/routes/students/controllers.js

```diff
@@ -24,3 +24,18 @@ export const getStudentProfile = async (req, res) => {
     return res.status(400).send("Could not get students");
   }
 };
+
+export const postPdSkills = async (req, res) => {
+  try {
+    let myPdPost = req.body;
+    !myPdPost.profile ? res.send('invalid data schema, need to add profile within POST!') : (pdSkillsPostData.push(myPdPost), res.status(200).json(myPdPost));
```

Comment:
What myPdPost stand for?

Reply:
Also remove empty space please

Reply:
myPdPost stands for the pd POST body which is being posted to the end point, renamed to pdPostBody

Reply:
empty space removed